shared/fs/container.js

```diff
@@ -26,7 +26,9 @@ const mapStateToProps = (state: TypedState, {routeProps}: OwnProps) => {
   }
```

Comment:
I know this isn't your code but 
1. we can use shorthand if the key and value are the same on line 25
```
path,
```
2. this mapping of items should happen in mergeProps and not mapStaetToProps